app/proguard.flags

```diff
@@ -0,0 +1,45 @@
+-keep class com.android.calendar.selectcalendars.SelectCalendarsSyncFragment
```

Comment:
you probably want to remove the `/proguard.flags` as well.

Reply:
..and rename the file here as well.

Reply:
Done